server/services/avatar-service.js

```diff
@@ -1,6 +1,15 @@
+const fs = require('fs');
+const path = require('path');
+
 module.exports = class UserService {
     saveAvatar = async (filename, customer) => {
         customer.avatar_img = filename;
         customer.save();
     };
+
+    getAvatar = (avatar_img) => {
+        const avatarPath = path.join(__dirname, `../../avatars/${avatar_img}`);
+        const base64Image = fs.readFileSync(avatarPath, 'base64');
+        return `data:image/jpeg;base64, ${base64Image}`;
+    };
 };
```

Comment:
You can make all methods of UserService `static`. Then you will not need create instances